.github/workflows/fast-forward.yml

```diff
@@ -0,0 +1,20 @@
+name: Fast forward merge
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  fast-forward:
+    if: ${{ github.event.label.name == 'fast-forward' && github.base_ref != 'main' }}
+    runs-on: ubuntu-22.04
+
+    steps:
```

Comment:
I think we need to remove the label as the first step

Reply:
done